packages/mobile/src/identity/verification.ts

```diff
@@ -209,23 +209,25 @@ interface AttestationsStatus {
 }
```

Comment:
Changes to this file just rewrite functions as generators, so that when they call functions that depend on zeroSyncMode, a selector can be used to read zeroSyncMode from state

Reply:
Note that I tested these changes to verification last week, before verification broke on alfajores